variants/vueapp/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This "maybe remove" comment applies to everything in cypress, I believe.

Reply:
cypress removed